Doc/library/http.server.rst

```diff
@@ -334,6 +334,10 @@ provides three different variants:
    .. versionchanged:: 3.9
       The *directory* parameter accepts a :term:`path-like object`.
 
+   .. versionchanged:: next
+      Added support for HTTP single-part range requests on files, as specified
+      in :rfc:`7233`.
```

Comment:
```suggestion
      in :rfc:`7233#section-2.1`.
```

I hope this renders well.

Reply:
RFC 9110 is published 2022 and is supposed to obsolete 7233.
https://datatracker.ietf.org/doc/html/rfc9110#name-range-requests